src/components/label.ts

```diff
@@ -54,15 +54,15 @@ export module Components {
     /**
      * Sets the Label's text.
      *
-     * @param {string} displayText
+     * @param {nubmer|string} displayText
      * @returns {Label} The calling Label.
      */
-    public text(displayText: string): Label;
-    public text(displayText?: string): any {
+    public text(displayText: number|string): Label;
```

Comment:
space before and after |
(we should probably have a TSLint rule for this, sorry)